packages/shared/src/components/Feed.tsx

```diff
@@ -146,6 +151,12 @@ export default function Feed<T>({
     isFetchingNextPage,
   } = usePostModalNavigation(items, fetchPage);
 
+  const { flags } = useContext(FeaturesContext);
+  const additionalInteractionButtonFeature = getFeatureValue(
+    Features.AdditionalInteractionButton,
+    flags,
+  );
```

Comment:
Since we have now merged the custom features context provider, I think we can now define the value for this feature in that context 🚢